r/configure.win

```diff
@@ -31,6 +31,12 @@ VERSION=$(grep ^Version DESCRIPTION | sed s/Version:\ //)
 # If binary not found, script exits nonzero
 if [ $? -ne 0 ]; then
   echo "Arrow C++ library was not found"
+else
```

Comment:
This looks like a bad merge with master; r/configure.win has changed since this branch was started. L46-49 below overwrites these.

Reply:
Right. I'll fix it.